src/core/image.cpp

```diff
@@ -11104,8 +11104,9 @@ void Image::Rotate2DInPlaceBy90Degrees(bool rotate_by_positive_90_degrees)
 {
 	// Rotate without interpolation by swapping indices.
```

Comment:
Maybe the function should be renamed if it also handles 3D? At least I would document in the comment that it also works for 3D.

```suggestion
	// Rotate without interpolation by swapping indices. For a 3D-volume the whole volume will be rotated around the Z-axis.
```

Reply:
I debated this, but the rotation itself is 2d (around a single axis) I'm more inclined to name after the action that object being acted on